tools/WebhookProcessor/github_webhook_processor.php

```diff
@@ -452,13 +452,12 @@ function get_pr_code_friendliness($payload, $oldbalance = null){
 		'Grammar and Formatting' => 1,
 		'Priority: High' => 4,
 		'Priority: CRITICAL' => 5,
-		'Atmospherics' => 4,
+		'Atmospherics' => 1,
```

Comment:
since atmos will always be tagged in conjunction with a superceding tag (tag worth more than one point or tag worth negative points) just remove it if you don't want it to be worth extra points. if you only want to nerf the value slightly then set it to 3.
as is it's effectively redundant